pkg/apis/hive/v1/vsphere/platform.go

```diff
@@ -0,0 +1,44 @@
+package vsphere
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Platform stores any global configuration used for vsphere platforms.
+type Platform struct {
+	// VCenter is the domain name or IP address of the vCenter.
+	VCenter string `json:"vCenter"`
+
+	// CredentialsSecretRef refers to a secret that contains the vSphere account access
+	// credentials: GOVC_HOST, GOVC_USERNAME, GOVC_PASSWORD fields.
```

Comment:
Why is GOVC_HOST required in the creds secret? Isn't that already covered by the `VCenter` field?

Reply:
It's not required. I'm using it as an alternative to providing the host as an argument to deprovision but the host can also be obtained from the cd.

Reply:
It can still be an alternative for the deprovision command, but that would be for a user calling deprovision directly. When Hive is deprovisioning, it must come from the ClusterDeployment.

Reply:
I still don't think that we should have the GOVC_HOST field in the secret. There should only be *one* place where the host comes from, and that is the `vCenter` field in the ClusterDeployment.

Reply:
:+1:  lets make that a runtime env var read in the deprovision code for a CLI user rather than a field in the secret.